openlibrary/templates/type/edition/view.html

```diff
@@ -258,6 +258,11 @@
                 <div>$_("Publish Date")</div>
                 <span itemprop="datePublished">$edition.publish_date</span>
               </div>
+            $if edition.original_publish_date:
+              <div class="edition-omniline-item">
```

Comment:
This can be moved closer to, maybe just above work.original_languages in this file.